spvc/README.asciidoc

```diff
@@ -0,0 +1,60 @@
+= spvc Manual
+:toc:
+:toclevels: 3
+:numbered:
+:source-highlighter: pygments
+
+== Name
+
+`spvc` - A command-line SPIR-V to GLSL/HLSL/MSL compiler with
+Clang-compatible arguments.
+
+== Synopsis
+
+----
+spvc [--help]
+
+spvc [--version]
+     [--validate=<target env>]
+     [-o outfile]
+     shader...
+----
+
+== Description
+
```

Comment:
I think it's useful here to describe the relationship with SPIRV-Cross.

Suggest

     spvc is a wrapper around SPIRV-Cross.   A SPIR-V module must pass validation before it is given to SPIRV-Cross for translation.

Reply:
done